rails/config/database.yml

```diff
@@ -10,12 +10,13 @@
 #   https://dev.mysql.com/doc/refman/5.7/en/password-hashing.html
 #
 default: &default
-  adapter: mysql2
+  adapter: postgresql
   encoding: utf8
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: root
```

Comment:
Pretty sure you need to provide a username/password for postgres. The documentation for the image you are using shows one being sent via -e POSTGRES_PASSWORD in the run command.